app/Http/Controllers/Store/BundleController.php

```diff
@@ -52,8 +51,7 @@ public function create(Registration $registration)
         $lastCollection = ($lastCollectedBundle && (!empty($lastCollectedBundle->disbursed_at)))
             // 'disbursed_at' is auto-carbon'd by the Bundle model
             ? $lastCollectedBundle->disbursed_at->format('l jS \of F Y')
-            : null
-        ;
+            : null;
```

Comment:
@charlesstrange2 Which do you prefer? The ARC codebase seems to have both.

Reply:
I genuinly have no preference generally. I tend to do it when i have multiline call chains
```
$thing->select()
  ->where('stuff')
  ->get()
  ->sortBy('sorting_thing')
;
```

Reply:
@charlesstrange2 do you want me to reverse these in the multiline call, i.e. tests especially in this PR?